app/models/studio.rb

```diff
@@ -1,3 +1,11 @@
 class Studio < ApplicationRecord
   has_many :movies
+
+  def studio_movies
+    movies.pluck(:title)
+  end
+
+  def contributing_actors
+    Actor.joins(:movies).where(movies: {id: movies}).distinct.pluck(:name)
```

Comment:
Great query! 👍 

Just for fun, how would you write this query if you relied instead on AR associations...? (spoiler/hint below)


First, you'd have to add 
```ruby
has_many :actors, through: :movies
```
to this model. At which point you could do: 
```ruby
actors.distinct #.pluck(:name) optionally
```

Either approach is fine!

Reply:
To be clear, the reason that extra `has_many` works is because this model already `has_many :movies`, so when we go to the Movie model, it already `has_many :actors`.... so, thanks to some transitive logic, a Studio would have many Actors through its Movies.